docs/cache.txt

```diff
@@ -0,0 +1,234 @@
+.. _laravel-queues:
+
+===============
+Cache and Locks
+===============
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, cache, lock, code example
+
+Configuration
+-------------
+
+In order to use MongoDB as backend for `Laravel Cache and Locks <https://laravel.com/docs/{+laravel-docs-version+}/cache>`__,
+add a store configuration using the ``mongodb`` driver in ``config/cache.php``:
+
+.. code-block:: php
+
+   'stores' => [
+       'mongodb' => [
+           'driver' => 'mongodb',
+           'connection' => 'mongodb',
+           'collection' => 'cache',
+           'lock_connection' => 'mongodb',
+           'lock_collection' => 'cache_locks',
+           'lock_lottery' => [2, 100],
+           'lock_timeout' => 86400,
+       ],
+   ],
+
+The following table describes a list of cache and lock options
+and their default values:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Setting
+     - Description
+
+   * - ``driver``
+     - **Required**. Specifies the lock driver to use. Must be ``mongodb``.
+
+   * - ``connection``
+     - **Required**. The database connection to use to store cache items. It must be a ``mongodb`` connection.
+
+   * - ``collection``
+     - Default ``cache``. Name of the MongoDB collection to store cache items.
+
+   * - ``lock_connection``
+     - Default to the cache ``connection``. The database connection to use to store locks. It must be a ``mongodb`` connection.
+
+   * - ``lock_collection``
+     - Default ``cache_locks``. Name of the MongoDB collection to store locks.
+
+   * - ``lock_lottery``
+     - Default ``[2, 100]``. Probability [chance, total] of pruning expired cache items. Set to [0, 0] to disable
+
+   * - ``lock_timeout``
+     - Default ``86400``. Time-to-live of the locks in seconds
+
+
+Setup auto-expiration indexes
+-----------------------------
+
+The :manual:`TTL indexes </core/index-ttl/>` integrated into MongoDB automatically
+delete documents when they have expired. Their use is optional with the ``mongodb``
+driver, but recommended as they provide better performance by delegating the
```

Comment:
https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-because

```suggestion
driver, but recommended, because they provide better performance by delegating the
```